packages/codemods/README.md

```diff
@@ -0,0 +1,43 @@
+# codemods
+
+A collection of codemods for Helios components.
+
+## Usage
```

Comment:
See comment below (sorry, I saw this after the comment; I don't approach the code review in linear way, at times)